mods/ts/rules/palettes.yaml

```diff
@@ -22,7 +22,6 @@
 		Name: chrome
 		Filename: cameo.pal
 		AllowModifiers: false
-		ShadowIndex: 242
```

Comment:
The easiest fix would be to specify the clock palette on the palette / support power widgets themselves.  There's no need for a per-icon clock sequence or clock sequence palette.